app/controllers/YourIncomeCalculationController.scala

```diff
@@ -108,7 +109,7 @@ class YourIncomeCalculationController @Inject()(
                 errorPagesHandler.internalServerError(
                   "Employment contains stub annual account data found meaning payment information can't be displayed")
               case (true, _) =>
-                Ok(views.html.print.historicIncomeCalculationView(historicIncomeCalculationViewModel, appConfig))
+                Ok(historicIncomeCalculationView(historicIncomeCalculationViewModel, appConfig))
```

Comment:
This is a different view, its actually the print view coming in, we should probs rename the view class to represent this

Reply:
Done